freqtrade/freqtradebot.py

```diff
@@ -657,6 +665,40 @@ def handle_trade(self, trade: Trade) -> bool:
         logger.info('Found no sell signals for whitelisted currencies. Trying again..')
         return False
 
+    def handle_stoploss_on_exchage(self, trade: Trade) -> bool:
```

Comment:
typo ... ? `handle_stoploss_on_exchange`

also i think this would deserve it's own test - as the method has quite some logic i would not want to see broken in furure changes ...